cmd/dc-supervisor/root.go

```diff
@@ -34,7 +34,7 @@ func init() {
 }
 
 // Starts cobra
-func Execute() {
+func main() {
 	if err := rootCmd.Execute(); err != nil {
 		fmt.Fprint(os.Stderr, err)
```

Comment:
May prefer the use of `FatalLn()` for command errors.

```suggestion
		fmt.FatalLn(os.Stderr, err)
```

Reply:
`log.Fatalln` equals `Println()` followed by `os.Exit(1)`.
We should either use `log.Fatalln` everywhere and remove`os.Exit(1)`, or keep it as is.

Reply:
or using `Errorln()`

```suggestion
		log.Errorln(err)
```

Reply:
Using `logrus` to print CLI related error is not wanted.